packages/x-data-grid/src/hooks/features/rowSelection/useGridRowSelection.ts

```diff
@@ -310,20 +302,36 @@ export const useGridRowSelection = (
   );
 
   const selectRows = React.useCallback<GridRowMultiSelectionApi['selectRows']>(
-    (ids: GridRowId[], isSelected = true, resetSelection = false) => {
+    (selectionModel: GridRowSelectionModel, isSelected = true, resetSelection = false) => {
```

Comment:
I wonder if leaving this as `selectRows(array)` would be better for DX than `selectRows(selectionModel)`.

Reply:
Yeah, I changed it back. Having both include/exclude model **and** `isSelected` param turned out to be too confusing.